install.sh

```diff
@@ -62,6 +62,13 @@ if [ "$RAM_AVAILABLE_IN_DOCKER" -lt "$MIN_RAM" ]; then
     exit -1
 fi
 
+#SSE4.2 required by Clickhouse (https://clickhouse.yandex/docs/en/operations/requirements/) 
+SSE_42_COMPAT="$(grep -c "sse4_2" /proc/cpuinfo)"
```

Comment:
This will fail on a macOS so I think we should do something similar to the memory detection command above:
```bash
SUPPORTS_SSE42=$(docker run --rm busybox grep -c sse4_2 /proc/cpuinfo);
```